src/dlgConnectionProfiles.cpp

```diff
@@ -133,8 +131,10 @@ void dlgConnectionProfiles::slot_update_description()
     if( pItem )
     {
         QString profile = pItem->text();
-        QString desc = mud_description_textedit->toPlainText();
-        writeProfileData( profile, QStringLiteral( "description" ), desc );
+        QString description = mud_description_textedit->toPlainText();
+        writeProfileData( profile, QStringLiteral( "description" ), description );
+
+        // don't display custom profile descriptions as a tooltip, as passwords could be stored in there
```

Comment:
Yeah, it will be shown when they explicitly click on it - and it won't be shown when they hover over it accidentally. I for example put passwords in there and wouldn't want them to be shown when I'm sharing my screen with someone else for example... maybe an impractical concern though.